doc/src/content/xdocs/spec.xml

```diff
@@ -1502,6 +1502,14 @@ void initFPTable() {
         <p>
           A <code>date</code> logical type annotates an Avro <code>int</code>, where the int stores the number of days from the unix epoch, 1 January 1970 (ISO calendar).
         </p>
+        <p>The following schema represents a date field:</p>
```

Comment:
Apologies in advance for being nit-picky!  This example is inconsistent with the example in decimal (which is a schema, while this is described as a schema snippet for a record field). 

I guess they're practically the same thing if you ignore the **name**, but I think it's worthwhile to be consistent.

Reply:
Thanks for the feedback. I noticed the same thing when adding name but did not have the background knowledge of the precise terminology.
I have updated the PR accordingly.